README.md

```diff
@@ -144,8 +146,8 @@ class Users extends MongoDataSource {
 
   updateUserName(userId, newName) {
     this.deleteFromCacheById(userId)
-    return this.collection.updateOne({ 
-      _id: userId 
```

Comment:
only saw this after my commit, i got my editor set up to remove trailing spaces automagically. unrelated ofc, but i guess they technically have no business being there ;)